### ISSUE TITLE
Different keyboard layouts

### ISSUE THREAD

**Issue by albfan** [akiraux/Akira#216]
## Expected Behavior
Zoom in shortcut <kbd>Ctrl</kbd>+<kbd>=</kbd> is not standard and breaks the workflow in spanish.

Typical zoom shorcuts are:
- <kbd>Ctrl</kbd>+<kbd>+</kbd>: Zoom in
- <kbd>Ctrl</kbd>+<kbd>-</kbd>: Zoom out
- <kbd>Ctrl</kbd>+<kbd>0</kbd>: Reset zoom

I check other programs (tilix https://github.com/gnunn1/tilix/blob/master/data/gsettings/com.gexperts.Tilix.gschema.xml#L1191) and this is pretty standard.

In spanish <kbd>Ctrl</kbd>+<kbd>+</kbd> is a direct key press. To use <kbd>Ctrl</kbd>+<kbd>=</kbd>, we need to press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>0</kbd>.

Seems in english is the opposite and <kbd>Ctrl</kbd>+<kbd>+</kbd> is not a direct keypress.

## Current Behavior

<kbd>Ctrl</kbd>+<kbd>=</kbd> executes zoom in.

## Possible Solution

- Stick to standards.
- Deal with this in a `l10n` way
- Allow to customize shortcuts

## Context
- Akira Version: master
- OS Info: Arch linux

**Comment by Alecaddd** [comment]
Ah! That makes sense, that's why we were colliding on that.
I Think for now we can stick to standards and then allow shortcut customization, so we don't need to handle localization for keyboard shortcuts, which it could get pretty hairy.
Thanks for investigating this.

**Comment by albfan** [comment]
Should we set to Ctrl +  + then?

**Comment by Alecaddd** [comment]
Yes, let's leave it like it is right now, after your update to master.
What do you think?

**Comment by albfan** [comment]
I think right now is Ctrl + =.

Any idea about how zoom in and out should behave? I think zoom in is not intuitive right now ( but can find how to make it more natural) Looking at other apps may work, but any idea is appreciated.

**Comment by Alecaddd** [comment]
You updated master here: https://github.com/akiraux/Akira/blob/master/src/Services/ActionManager.vala#L89
So it's `plus`, let's leave it like that for now.

For the zoom, I'm taking care of that in the `canvas-action` branch, I should have a PR ready for review tomorrow.
Here's the behaviour:

Zoom in: <kbd>CTRL</kbd> + <kbd>Scroll Up</kbd>
Zoom out: Zoom in: <kbd>CTRL</kbd> + <kbd>Scroll down</kbd>
